client/src/components/App.js

```diff
@@ -31,14 +31,12 @@ class App extends React.Component {
     this.setPlaylist()
   }
 
-  setPlaylist = () => {
-    const { getPlaylist } = this.props
+  onClickLogout = () => this.props.logout()
```

Comment:
We prefer object destructuring in cases like this. Primarily to improve code readability, and also to decrease code repetition. You'll say I have only one line, yes you have only one line, but code consistency is also an important thing to obey. 
So I would prefer something like this
```javascript
onClickLogout = () => {
  const {logout} = this.props;

  logout();
}
```